storage/rocksdb/CMakeLists.txt

```diff
@@ -119,18 +119,14 @@ STRING(REGEX MATCHALL "[^\n]+" ROCKSDB_LIB_SOURCES ${SCRIPT_OUTPUT})
 # add bundled compression code
 SET(ROCKSDB_LIB_SOURCES
   ${ROCKSDB_LIB_SOURCES}
-  ${CMAKE_CURRENT_SOURCE_DIR}/third_party/lz4/lib/lz4.c
-  ${CMAKE_CURRENT_SOURCE_DIR}/third_party/lz4/lib/lz4hc.c
-  ${CMAKE_CURRENT_SOURCE_DIR}/third_party/lz4/lib/lz4frame.c
-  ${CMAKE_CURRENT_SOURCE_DIR}/third_party/lz4/lib/xxhash.c
+  ${CMAKE_SOURCE_DIR}/extra/lz4/xxhash.c
```

Comment:
Why is this different than in 5.7?

Reply:
In 5.7 we are using XXHash from `zstd` submodule with:
```
${CMAKE_CURRENT_SOURCE_DIR}/third_party/zstd/lib/common/xxhash.c
```
In 8.0 we removed `zstd` submodule.

The second option for 8.0 is using `extra/zstd/lib/common/xxhash.c` (XXHash 0.6.2) but `extra/lz4/xxhash.c` is newer with XXHash 0.6.5.